### ISSUE TITLE
metabase.util.malli.fn Invalid input - Please report this as an issue on Github

### ISSUE THREAD

**Issue by Thossine** [metabase/metabase#38096]
**Describe the bug**
I have a ton of logs that ask me to create a ticket. Below a partial portion of the logs generated.

**Logs**

```
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name query-hash}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name query-hash}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name select-keys-for-hashing}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name query-hash}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name select-keys-for-hashing}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name select-keys-for-hashing}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name query-hash}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name select-keys-for-hashing}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name query-hash}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name select-keys-for-hashing}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name query-hash}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name select-keys-for-hashing}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name query-hash}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name query-hash}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name select-keys-for-hashing}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name select-keys-for-hashing}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name query-hash}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name query-hash}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name select-keys-for-hashing}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name query-hash}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name select-keys-for-hashing}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name query-hash}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name select-keys-for-hashing}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name query-hash}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name select-keys-for-hashing}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name query-hash}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name select-keys-for-hashing}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name select-keys-for-hashing}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name query-hash}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name select-keys-for-hashing}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name query-hash}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name select-keys-for-hashing}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name query-hash}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:54+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name select-keys-for-hashing}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:55+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name query-hash}
[7c45fff6-2e39-447d-9e7b-b62432db60a0] 2024-01-24T16:27:55+01:00 WARN metabase.util.malli.fn Invalid input - Please report this as an issue on Github: ["invalid type"] {:type :metabase.util.malli.fn/invalid-input, :error {:schema :map, :value nil, :errors ({:path [], :in [], :schema :map, :value nil, :type :malli.core/invalid-type})}, :humanized [invalid type], :schema :map, :value nil, :fn-name query-hash}
```

**To Reproduce**
I'm calling an embedded dashboard and passing a lot of filter value into the JWT token. 

**Expected behavior**
Some WARN are generated into the syslogs

**Screenshots**
N/A

**Severity**
Metabase seems to slowly slow down over time until i restart it. Does the log over feeding at some point, making metabase slower ?
Anyway, it isn't that much of pain, i have a crontab that restart sever everynight to workaround this.

**Additional context**
N/A

**Metabase Diagnostic Info**

```json
{
  "browser-info": {
    "language": "fr-FR",
    "platform": "Win32",
    "userAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36",
    "vendor": "Google Inc."
  },
  "system-info": {
    "file.encoding": "UTF-8",
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "11.0.21+9-post-Debian-1deb10u1",
    "java.vendor": "Debian",
    "java.vendor.url": "https://tracker.debian.org/openjdk-11",
    "java.version": "11.0.21",
    "java.vm.name": "OpenJDK 64-Bit Server VM",
    "java.vm.version": "11.0.21+9-post-Debian-1deb10u1",
    "os.name": "Linux",
    "os.version": "4.19.0-26-amd64",
    "user.language": "fr",
    "user.timezone": "Europe/Paris"
  },
  "metabase-info": {
    "databases": [
      "mysql",
      "postgres",
      "sqlserver",
      "oracle"
    ],
    "hosting-env": "unknown",
    "application-database": "h2",
    "application-database-details": {
      "database": {
        "name": "H2",
        "version": "2.1.214 (2022-06-13)"
      },
      "jdbc-driver": {
        "name": "H2 JDBC Driver",
        "version": "2.1.214 (2022-06-13)"
      }
    },
    "run-mode": "prod",
    "version": {
      "date": "2024-01-11",
      "tag": "v0.48.3",
      "hash": "80d8323"
    },
    "settings": {
      "report-timezone": "Europe/Paris"
    }
  }
}
```

**Comment by WangDanpeng** [comment]
I'm also encountering the same issue. Have you found a solution?

**Comment by camsaul** [comment]
This was fixed in 49 or 50 in #39969 

https://github.com/metabase/metabase/pull/39969/files#diff-75c200b87b45853b63d1c964a01786998b5049409ee07a8887a6a26496fa618bL113-R113

and in 48.6 by #38482

**Comment by camsaul** [comment]
Fixed by #38482
